docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,533 @@
-possible. A node might be unrepairable and does not start-up,
-so if the cluster is still available the node can be simply replaced by
-a fresh node on a new host which will replace the broken node.
+.. rubric:: Table of contents
 
-crate-node CLI
-==============
+.. toctree::
+   :maxdepth: 1
 
-.. code-block:: console
+.. contents::
+   :local:
 
-   bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
-   [--ordinal <Integer>] [-C <KeyValuePair>]
-   [-h, --help] ([-s, --silent] | [-v, --verbose])
+Changing the role of a node
+===========================
 
+Each CrateDB data node maintains the following data on disk:
 
-The following commands provide a various operations to modify and recover nodes:
+* the shard data for every shard allocated to that node,
+* the index metadata corresponding with every shard allocated to that node, and
+* the cluster-wide metadata, such as settings and index templates.
 
-* ``crate-node repurpose`` can be used to delete unwanted data from a node if it
-  used to be a data node or a master-eligible node but has been repurposed not to
-  have one or other of these roles.
+Similarly, each CrateDB  master-eligible node maintains the following data on
+disk:
 
-* ``crate-node unsafe-bootstrap`` can be used to perform unsafe cluster bootstrapping.
-  It forces one of the nodes to form a brand-new cluster on its own, using its local
-  copy of the cluster metadata.
+* the index metadata for every index in the cluster, and
+* the cluster-wide metadata, such as settings and index templates.
 
-* ``crate-node detach-cluster`` enables you to move nodes from one cluster to another.
-  This can be used to move nodes into a new cluster created with the
-  ``crate-node unsafe-bootstap`` command. If unsafe cluster bootstrapping was not
-  possible, it also enables you to move nodes into a brand-new cluster.
+Now, the `repurpose command`_ lets you delete any of a node's shard data,
+repurpose the node and restart the node afterward.
 
-Changing the role of a node
----------------------------
+Before running the ``repurpose`` command
+----------------------------------------
 
-Certain situations may require to change the initial type of a node. The
-``crate-node repurpose`` command can modify and delete the data stored on a  disk
-from a node.
+Adhere to the following procedure to change the role of a node:
 
-The intended use is:
+* Step 1: Stop the node
+* Step 3: Update `crate.yml` by setting ``node.master`` and ``node.data`` as
+  desired
+* Step 4: Run ``crate-node repurpose`` on the node
+* Step 5: Restart the node
 
-* Stop the node
+Certain situations may require you to change the initial type of a node by
+modifying and deleting its on-disk data – for example, if you have
+irrecoverably lost the majority of the master-eligible nodes and you need to
+form a new cluster.
 
-* Update crate.yml by setting node.master and node.data as desired.
+Running the ``repurpose`` command
+---------------------------------
```

Comment:
at the start of this section, it might be nice to explain/indicate that as there are three possible combinations of valid configuration settings, that corresponds to three repurposing scenarios

Reply:
in fact, thinking about it: perhaps it makes sense to replace these subsections with a table that show all possible combinations of these two config values and states whether they're valid (three of them are) and has a prose description of why you might want to do that

that will cut down on a lot of duplication in this section